x-pack/legacy/plugins/graph/public/angular/templates/index.html

```diff
@@ -156,7 +159,15 @@
               <circle class="gphNode__circle " r="10" cx="12" cy="12" ng-attr-style="fill:{{n.color}}"
                 ng-click="workspace.deselectNode(n)" />
 
-              <text ng-if="n.icon" class="fa gphNode__text" text-anchor="middle" x="12" y="17" ng-click="workspace.deselectNode(n)">{{n.icon.code}}</text>
+              <text
+                ng-if="n.icon"
+                class="fa gphNode__text gphSelectionList__icon"
+                text-anchor="middle"
+                x="12"
+                y="16"
+                ng-click="workspace.deselectNode(n)"
+                ng-class="{'gphNode__text--inverse': isColorDark(n.color)}"
```

Comment:
Adding the additional class to change the icon color if the vertex color becomes too dark